tests/package-title-dir-rtl-001/OPS/content_001.xhtml

```diff
@@ -1,13 +1,18 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
```

Comment:
Optional, but I'd recommend putting this new test in a separate PR — unrelated. I won't review this test carefully in this PR.

Reply:
You are right. Let us play by the rules and add these via a separate PR. Will do.